src/lib/fragments/ExpandableMenuItem.react.js

```diff
@@ -31,7 +32,7 @@ class ExpandableMenuItem extends Component {
         <a className={titleClass} href="#" data-expandable-control="expandable-link">
           {title}
         </a>
-        <div id="expandable-menu-contents" hidden={this.state.hidden}>
+        <div id={"expandable-menu-contents-" + uuid()} hidden={this.state.hidden}>
```

Comment:
Two things, is there a valid reason for this being an ID. Also if there is, then have you considered appending the prop ID to this instead of UUID?

Reply:
Thanks Jordan. It needed a unique ID as when we were running accessibility checks we were getting a warning about non-unique ID's. Will take a look into using prop ID instead of UUID.